### PR TITLE
Multiclass Classification tutorial

### PR DESCRIPTION
**Pull request by JRAlexander**
GitHub Issue Multiclass Classification tutorial

[Internal Review URL](https://review.docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/github-issue-classification?branch=pr-en-us-10038)

### CODE REVIEW

File: docs/machine-learning/tutorials/github-issue-classification.md [hunk 1]
```diff
@@ -0,0 +1,474 @@
...
+description: Discover how to use ML.NET in a multi classification scenario to classify GitHGub issues to assign them to a given area.
+ms.date: 01/18/2018
+ms.topic: tutorial
+ms.custom: mvc
+#Customer intent: As a developer, I want to use ML.NET to apply a multi classification task so that I can understand how to classify GitHGub issues to assign them to a given area.
+---
+# Tutorial: Use ML.NET in a multi classification scenario to classify GitHub issues.
+
+> [!NOTE]
+> This topic refers to ML.NET, which is currently in Preview, and material may be subject to change. For more information, visit [the ML.NET introduction](https://www.microsoft.com/net/learn/apps/machine-learning-and-ai/ml-dotnet).
```
[the remark this answers is not in the document]
Comment (JRAlexander) [reply docs/machine-learning/tutorials/github-issue-classification.md]: Good idea!

[the remark this answers is not in the document]
Comment (JRAlexander) [reply docs/machine-learning/tutorials/github-issue-classification.md]: So, you aren't reviewing, just commenting?

**Review by CESARDELATORRE — APPROVED** [review]

**Review by Thraka — APPROVED** [review]
Approve for when you get my fixes in (if you want them)

**Comment by JRAlexander** [comment]
Addressed changes. Thanks, @Thraka and @CESARDELATORRE

**Pull request merged**